lib/internal/per_context/domexception.js

```diff
@@ -49,12 +49,23 @@ const disusedNamesSet = new SafeSet()
   .add('ValidationError');
 
 class DOMException {
-  constructor(message = '', name = 'Error') {
+  constructor(message = '', options = 'Error') {
     ErrorCaptureStackTrace(this);
-    internalsMap.set(this, {
-      message: `${message}`,
-      name: `${name}`
-    });
+
+    if (options && typeof options === 'object') {
+      const { name, cause } = options;
+      internalsMap.set(this, {
+        message: `${message}`,
+        name: `${name}`
+      });
+
+      ObjectDefineProperty(this, 'cause', { __proto__: null, value: cause });
```

Comment:
Shouldn't we set the `cause` only if there was one passed?
```suggestion
      if ('cause' in options) ObjectDefineProperty(this, 'cause', { __proto__: null, value: cause });
```

Reply:
Oh yeah, thank you!